ui/js/models/perf/alerts.js

```diff
@@ -151,19 +151,34 @@ treeherder.factory('PhAlerts', [
             return _.find(phAlertSummaryStatusMap, { id: this.status }).text;
         };
 
+        function _getAlertSummary(id) {
+            // get a specific alert summary
+            var httpTimeout = $q.defer();
+            var request = ThOptionCollectionModel.getMap();
+            var promise = request.then(
```

Comment:
No need for `request` to be a seperate variable here. I also don't understand the purpose of $q.defer() and httpTimeout?